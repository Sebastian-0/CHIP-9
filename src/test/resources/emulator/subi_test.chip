ldi A, 0

# Carry and Zero flags (should carry really be set here?)
subi 0
sout FLAGS
sout A

# Carry and Negative
subi 1
sout FLAGS
sout A

# Half-carry flag
ldi A, 2
subi 1
sout FLAGS
sout A

hcf