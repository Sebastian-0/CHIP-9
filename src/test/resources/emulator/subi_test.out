144
0

80
255

32
1